storage/invoke_test.go

```diff
@@ -171,16 +174,34 @@ func TestInvoke(t *testing.T) {
 				}
 				return test.finalErr
 			}
-			got := run(ctx, call, test.retry, test.isIdempotentValue)
+			req := &fakeApiaryRequest{header: http.Header{}}
+			got := run(ctx, call, test.retry, test.isIdempotentValue, setRetryHeaderHTTP(req))
 			if test.expectFinalErr && got != test.finalErr {
 				s.Errorf("got %v, want %v", got, test.finalErr)
 			} else if !test.expectFinalErr && got != test.initialErr {
 				s.Errorf("got %v, want %v", got, test.initialErr)
 			}
+			wantHeader := fmt.Sprintf("gccl-invocation-id/.{36} gccl-attempt-count/%v gl-go/.* gccl/", counter)
```

Comment:
More so a question of my understanding... just wanted to verify that this test does test a retry and that the invocation ID remains consistent?

Reply:
This just checks that the header has the expected format and that the attempt count is the expected number of retries. It doesn't specifically test that the ID remains the same between retries.

I guess I could write a separate test that checks that the ID remains the same.

Reply:
Figured out how to make the original test capture the original UUID and make sure that it's consistent.